src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -48,5 +54,18 @@ export default {
 			return this.userInfo?.firstName ?? '';
 		}
 	},
+	methods: {
+		async getRecommendedLoans() {
+			const { loans } = await runLoansQuery(
+				this.apollo,
+				{ sortBy: 'personalized', pageLimit: 6 },
+				FLSS_ORIGIN_NOT_SPECIFIED
+			);
+			this.recommendedLoans = loans;
+		}
+	},
+	created() {
```

Comment:
Do we want this on created or mounted?

Reply:
To get the loans data we don't need access to the DOM so its okay to have this on created I think.

Reply:
If it's in created it will kick off on server render, if that is the intent, please very by loading the page locally without js to see if this call is completing. I think it "should" complete b/c you have the async/await combo. If it doesn't complete you'll prob need to tweak things.

Reply:
Or just move it to mounted skip the extra initial server load.

Reply:
Yeah because it's not being awaited, I'm not sure when it officially completes, like if the server render waits (probably not). Mounted is probably the more efficient route, if we're okay with an initial loading state.

Reply:
ok cool! lets move it to mounted. thanks guys!